### PR TITLE
Remove mobile views

### PR DESCRIPTION
**Pull request by Georgehe4**
Remove mobile views from DartPad.

Code changes pushed to http://dartpad-test.appspot.com/

### CODE REVIEW

**Comment by kwalrath** [comment]
Can you tell me more about this? What does this removal mean for how DartPad will work on phones?

**Comment by devoncarew** [comment]
Thanks for the PR! I may have a slightly delayed review, but will take a look shortly.

> Can you tell me more about this? What does this removal mean for how DartPad will work on phones?

We've discussed this off and on. We have 3 separate UIs for dartpad, and the associated maintenance burden. The mobile UI is the least used - and in terms of consolidating the different UIs - the most likely one to remove.

Without looking at the PR, we'd want to make sure that the appengine re-direct (sending mobile devices to the mobile UI) is removed, and that the css with the main UI is ~responsive, in that we know how it would look on mobile and have done some work to improve from the default.

File: tool/grind.dart [hunk 1]
```diff
@@ -81,14 +78,13 @@ build() {
...
   vulcanizeNoExclusion('scripts/imports.html');
-  vulcanize('mobile.html');
   vulcanize('index.html');
   vulcanize('embed-dart.html');
   vulcanize('embed-html.html');
   vulcanize('embed-inline.html');
 
   return _uploadCompiledStats(
-      mainFile.asFile.lengthSync(), mobileFile.asFile.lengthSync());
+      mainFile.asFile.lengthSync());
```
Comment (devoncarew) [inline tool/grind.dart]: Let's dartfmt this file.

**Comment by devoncarew** [comment]
And, sorry for the long delay :)

**Comment by devoncarew** [comment]
Fantastic, thanks!

**Comment by Georgehe4** [comment]
Test instance can be found at http://dartpad-test.appspot.com/

Sample ss on mobile:
screen shot 2018-02-02 at 6 21 26 pm

**Comment by kwalrath** [comment]
I'll adjust the DartPad page accordingly (removing a special case for the mobile UI is the only change, I think).

**Comment by devoncarew** [comment]
@Georgehe4, did we publish this?

**Comment by Georgehe4** [comment]
These changes are now pushed to prod.

**Comment by devoncarew** [comment]
> These changes are now pushed to prod.

Awesome, thanks!

**Comment by kwalrath** [comment]
Docs were updated in https://github.com/dart-lang/site-www/pull/555/commits/f96cf90e37da7649e70ec4dc298d59c726b232ab

**Pull request merged**